opencti-platform/opencti-graphql/src/domain/location.js

```diff
@@ -13,7 +14,7 @@ export const findById = async (context, user, locationId) => {
 export const findAll = async (context, user, args) => {
   let types = [];
   if (args.types && args.types.length > 0) {
-    types = filter((type) => isStixDomainObjectLocation(type), args.types);
+    types = R.filter((type) => isStixDomainObjectLocation(type), args.types);
```

Comment:
Please use standard JS filter function instead of Ramda

Reply:
ok